tests/message/unittest_message_id_store.py

```diff
@@ -90,3 +94,47 @@ def test_duplicate_msgid(msgid_store: MessageIdStore) -> None:
         "Message symbol 'warning-symbol' cannot be used for 'W1234' and 'W1235'"
         in str(error.value)
     )
+
+
+def test_exclusivity_of_msg_ids() -> None:
+    """Test to see if all checkers have an exclusive message id prefix"""
+    err_msg = (
+        "{} has the same prefix ('{}') as the '{}' checker. Please make sure the prefix "
+        + "is unique for each checker. You can use 'script/get_unused_message_id_category.py' "
+        + "to get an unique id."
```

Comment:
```suggestion
        "{} has the same prefix ('{}') as the '{}' checker. Please make sure the prefix "
        "is unique for each checker. You can use 'script/get_unused_message_id_category.py' "
        "to get an unique id."
```
Implicit concatenation of string would work.